tailored-css.md

```diff
@@ -0,0 +1,120 @@
+# Tailored CSS
+
+## The site should be fast
+
+We value site speed a lot at SpanishDict.com. As we've started expanding our
+site into Latin America, this has become even more important. Connection speeds
+tend to be slower than in the USA, so optimizing for them is always on our mind.
+
+In researching how to improve our site's speed, a recurring theme is that
+[perceived speed tends to be more important than actual speed](http://blog.teamtreehouse.com/perceived-performance).
+We're always on the lookout for ways to make the site _feel_ faster.
+
+## State of CSS at SpanishDict
+
+One of the things that can slow down how a site feels (it certainly did for us)
+is blocking CSS. When you add a `link` tag, the browser has to first load and
+parse the entire file, because it doesn't know how to render the content without
```

Comment:
Here's a link to cite about `link` being blocking, if you want: https://jakearchibald.com/2016/link-in-body/